### ISSUE TITLE
[BUG] LSPosed is not currently installed or active.

### ISSUE THREAD

**Issue by Rtizer-9** [LSPosed/LSPosed#545]
**什么情况/What happened?**

 LSPosed Manager shows "LSPosed is not currently installed or active" everytime I activate a module and reboot. It stays activated if no module is activated. I tried with both debug and stable build.

**Xposed模块列表/Xposed Module List**
 XPrivacyLua, DisableFlagSecure, LuckyPatcher, JasiPatcher

**Magisk模块列表/Magisk Module List**
ADB & Fastboot for Andoird NDK, Ainur Narsil, ANXCamera, ANXCamera RN10P addon, App Systemizer, Audio Modification Library, Busybox for Android NDK, Debloater, Dolby Digital Plus, JamesDSP manager, Magisk Manager for Recovery Mode, Move Certificates, Riru, Riru -LSPosed, Webview Manager

**版本信息/Versions**

Android:11

Magisk:22104 (Canary)

Riru:25.4.3

LSPosed:1.3.5-5601 Debug & Release

**相关日志/Related Logs**

 
[boot-2021-05-07-09-21-46.log](https://github.com/LSPosed/LSPosed/files/6440588/boot-2021-05-07-09-21-46.log) (magiskhide is already turned off as you suggested in a bug)
This is captured with Log Catcher v22

I can't capture any log from LSPosed manager as it only shows log when LSPosed is active.

**Comment by yujincheng08** [comment]
This problem is caused by Riru.

Your zygote died during startup and Riru fails to load itself again. Please submit an issue to Riru instead.

https://github.com/RikkaApps/Riru

**Comment by vvb2060** [comment]
```
05-07 14:51:19.689  2237  2237 I rirud_java: Riru loaded, stop rirud socket...
05-07 14:51:19.874  2237  2349 W JavaBinder: BinderProxy is being destroyed but the application did not call unlinkToDeath to unlink all of its death recipients beforehand.  Releasing leaked death recipient: a.a
05-07 14:51:19.874  2237  2349 I BpBinder: onLastStrongRef automatically unlinking death recipients: <uncached descriptor>
```

**Comment by yujincheng08** [comment]
try this riru
[riru-v25.4.4-release.zip](https://github.com/LSPosed/LSPosed/files/6441050/riru-v25.4.4-release.zip)

**Comment by Rtizer-9** [comment]
Ok, Let me try.

**Comment by yujincheng08** [comment]
Then u should fire an new issue with log.

**Comment by Rtizer-9** [comment]
Here, or on Riru?

**Comment by yujincheng08** [comment]
Here

**Comment by Rtizer-9** [comment]
My phone is in recovery now due to Rescue Party trigger and I can see boot.log in cache folder. Is this the log you want?
Because in order to boot, I'll have to remove the LSPosed module and then the log wouldn't be of any use to you after that removal.

**Comment by aviraxp** [comment]
@Rtizer-9 Yes, that's it

**Comment by Rtizer-9** [comment]
[boot.log](https://github.com/LSPosed/LSPosed/files/6441622/boot.log)
Riru: 25.4.4
Magisk: 22104 Canary
LSPosed: 1.3.5 (5601) debug

Reboot after every flash.
Phone boots fine if no module is activated.

I then enabled just xprivacylua and only selected the recommended scope selected by default. Now, phone goes to recovery "android rescue party trigger" after the reboot.

For test I have disabled every other module in magisk except Log Catcher and Magisk Manager for Recovery Mode (for recovering from bootloop.
